src/openrct2-ui/input/KeyboardShortcuts.cpp

```diff
@@ -305,4 +305,6 @@ const uint16 KeyboardShortcuts::DefaultKeys[SHORTCUT_COUNT] =
     PLATFORM_MODIFIER | SDL_SCANCODE_L,         // SHORTCUT_LOAD_GAME
     SDL_SCANCODE_B,                             // SHORTCUT_CLEAR_SCENERY
     SDL_SCANCODE_7,                             // SHORTCUT_GRIDLINES_DISPLAY_TOGGLE
+    SDL_SCANCODE_U,                             // SHORTCUT_VIEW_CLIPPING
```

Comment:
I don't think this should have a shortcut key by default. It's a tool for more advanced users and not for casual gameplay.